doc/Makefile

```diff
@@ -5,7 +5,8 @@ default: html
 # You can set these variables from the command line.
 SRCDIR           := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 JULIAHOME        := $(abspath $(SRCDIR)/..)
-JULIA_EXECUTABLE := $(JULIAHOME)/usr/bin/julia
+include $(JULIAHOME)/Make.inc
```

Comment:
I'm coming up with some workarounds for the windows buildbot issues where it turns out it would be useful to have included Make.inc for in this file, so merging.